src/internal/Subscriber.ts

```diff
@@ -132,46 +132,40 @@ export class SafeSubscriber<T> extends Subscriber<T> {
   ) {
     super();
 
-    // If we don't have arguments, or the observer passed is already EMPTY_OBSERVER,
-    // use EMPTY_OBSERVER. This is just to save a little on object allocations.
-    this.destination = EMPTY_OBSERVER;
-    if ((observerOrNext || error || complete) && observerOrNext !== EMPTY_OBSERVER) {
```

Comment:
After some thought, I realized that this check, and setting `this.destination = EMPTY_OBSERVER` was pointless, so that got removed.

It might be better to review this with "ignore whitespace" on.